### PR TITLE
[zh-cn] Resync container-runtimes page

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/zh-cn/docs/setup/production-environment/container-runtimes.md [hunk 1]
```diff
@@ -112,28 +116,6 @@ In order for a Linux node's iptables to correctly view bridged traffic, verify t
 为了让 Linux 节点的 iptables 能够正确查看桥接流量，请确认 `sysctl` 配置中的
 `net.bridge.bridge-nf-call-iptables` 设置为 1。 例如：
```
Comment (Sea-n) [inline content/zh-cn/docs/setup/production-environment/container-runtimes.md:117]:
```suggestion
`net.bridge.bridge-nf-call-iptables` 设置为 1。例如：
```

File: content/zh-cn/docs/setup/production-environment/container-runtimes.md [hunk 2]
```diff
@@ -388,33 +393,53 @@ When using kubeadm, manually configure the
...
+CRI-O uses the systemd cgroup driver per default, which is likely to work fine
+for you. To switch to the `cgroupfs` cgroup driver, either edit
+`/etc/crio/crio.conf` or place a drop-in configuration in
+`/etc/crio/crio.conf.d/02-cgroup-manager.conf`, for example: 
+-->
+#### cgroup 驱动程序   {#cgroup-driver}
 
+CRI-O 默认使用 systemd cgroup 驱动程序，这对你来说可能工作得很好。
+要切换到 `cgroupfs` cgroup 驱动程序，请编辑 `/etc/crio/crio.conf` 或在
+`/etc/crio/crio.conf.d/02-cgroup-manager.conf` 中放置一个插入式配置 ，例如：
```
Comment (Sea-n) [inline content/zh-cn/docs/setup/production-environment/container-runtimes.md:442]:
```suggestion
`/etc/crio/crio.conf.d/02-cgroup-manager.conf` 中放置一个插入式配置，例如：
```

File: content/zh-cn/docs/setup/production-environment/container-runtimes.md [hunk 3]
```diff
@@ -199,23 +183,24 @@ stabilized the system. To configure this for Docker, set `native.cgroupdriver=sy
 更改设置，令容器运行时和 kubelet 使用 `systemd` 作为 cgroup 驱动，以此使系统更为稳定。
 对于 Docker, 设置 `native.cgroupdriver=systemd` 选项。
```
Comment (Sea-n) [inline content/zh-cn/docs/setup/production-environment/container-runtimes.md:184]:
```suggestion
对于 Docker，设置 `native.cgroupdriver=systemd` 选项。
```

File: content/zh-cn/docs/setup/production-environment/container-runtimes.md [hunk 4]
```diff
@@ -82,8 +88,7 @@ The following steps apply common settings for Kubernetes nodes on Linux.
 You can skip a particular setting if you're certain you don't need it.
 
 For more information, see [Network Plugin Requirements](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#network-plugin-requirements) or the documentation for your specific container runtime.
- -->
-
+-->
 ## 安装和配置先决条件
```
Comment (Sea-n) [inline content/zh-cn/docs/setup/production-environment/container-runtimes.md:92]:
```suggestion
## 安装和配置先决条件   {#install-and-configure-prerequisites}
```

File: content/zh-cn/docs/setup/production-environment/container-runtimes.md [hunk 5]
```diff
@@ -102,7 +107,6 @@ To load it explicitly, run `sudo modprobe br_netfilter`.
 
 In order for a Linux node's iptables to correctly view bridged traffic, verify that `net.bridge.bridge-nf-call-iptables` is set to 1 in your `sysctl` config. For example: 
 -->
-
 ### 转发 IPv4 并让 iptables 看到桥接流量
```
Comment (Sea-n) [inline content/zh-cn/docs/setup/production-environment/container-runtimes.md:110]:
```suggestion
### 转发 IPv4 并让 iptables 看到桥接流量   {#forwarding-ipv4-and-letting-iptables-see-bridged-traffic}
```

File: content/zh-cn/docs/setup/production-environment/container-runtimes.md [hunk 6]
```diff
@@ -156,21 +138,23 @@ sudo sysctl --system
 
 <!--
 ## Cgroup drivers
+
+On Linux, {{< glossary_tooltip text="control groups" term_id="cgroup" >}}
+are used to constrain resources that are allocated to processes.
 -->
 ## Cgroup 驱动程序
```
Comment (Sea-n) [inline content/zh-cn/docs/setup/production-environment/container-runtimes.md:145]:
```suggestion
## cgroup 驱动程序   {#cgroup-drivers}
```

File: content/zh-cn/docs/setup/production-environment/container-runtimes.md [hunk 7]
```diff
@@ -292,21 +278,24 @@ In order to use it, cgroup v2 must be supported by the CRI runtime as well.
 
 <!-- 
 ### Migrating to the `systemd` driver in kubeadm managed clusters
+
+If you wish to migrate to the `systemd` cgroup driver in existing kubeadm managed clusters,
+follow [configuring a cgroup driver](/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver/).
 -->
 ### 将 kubeadm 托管的集群迁移到 `systemd` 驱动
```
Comment (Sea-n) [inline content/zh-cn/docs/setup/production-environment/container-runtimes.md:285]:
```suggestion
### 将 kubeadm 托管的集群迁移到 `systemd` 驱动   {#migrating-to-the-systemd-driver-in-kubeadm-managed-clusters}
```

**Review by Sea-n — COMMENTED** [review]
/lgtm

File: content/zh-cn/docs/setup/production-environment/container-runtimes.md (hunk 5 above)
Comment (Sea-n) [reply content/zh-cn/docs/setup/production-environment/container-runtimes.md:110]: Okay, I know it's not necessary to add link anchors for h3 like this now.

File: content/zh-cn/docs/setup/production-environment/container-runtimes.md [hunk 8]
```diff
@@ -44,36 +46,40 @@ Kubernetes {{< skew currentVersion >}} 要求你使用符合{{<glossary_tooltip
...
+
 {{< note >}}
+<!-- 
 Kubernetes releases before v1.24 included a direct integration with Docker Engine,
 using a component named _dockershim_. That special direct integration is no longer
 part of Kubernetes (this removal was
 [announced](/blog/2020/12/08/kubernetes-1-20-release-announcement/#dockershim-deprecation)
 as part of the v1.20 release).
+-->
+v1.24 之前的 Kubernetes 版本包括与 Docker Engine 的直接集成，使用名为 _dockershim_ 的组件。
```
Comment (howieyuen) [inline content/zh-cn/docs/setup/production-environment/container-runtimes.md:58]:
```suggestion
v1.24 之前的 Kubernetes 版本包括与 Docker Engine 的直接集成，使用名为 **dockershim** 的组件。 
```

**Review by howieyuen — COMMENTED** [review]
one nit

**Pull request merged**